tools/quality.gradle

```diff
@@ -8,6 +8,5 @@ task ktlint(type: JavaExec, group: "verification") {
     main = "com.github.shyiko.ktlint.Main"
     args "src/**/*.kt"
     args "--android=true"
-    args "--ruleset=${rootDir}/tools/html-reporter.jar"
-    args "--reporter=html,output=${buildDir}/ktlint.html"
+    args "--reporter=html,artifact=com.github.mcassiano:ktlint-html-reporter:0.0.1,output=${buildDir}/ktlint.html"
```

Comment:
It's this "com.github.mcassiano:ktlint-html-reporter:0.0.1" correct? Shouldn't be from the main repo?

Reply:
It’s correct, see https://github.com/mcassiano/ktlint-html-reporter

Reply:
Ok, LGTM